vertical-pod-autoscaler/deploy/vpa-v1-crd-gen.yaml

```diff
@@ -229,7 +236,23 @@ spec:
     singular: verticalpodautoscaler
   scope: Namespaced
   versions:
-  - name: v1
+  - additionalPrinterColumns:
```

Comment:
Can you provide some sample output?

Reply:
Sure, this is the output of sample apps.
```
$ kubectl get vpa
NAME          MODE   CPU    MEM       PROVIDED   AGE
hamster-vpa   Auto   587m   262144k   True       2m49s
redis-vpa     Auto   25m    262144k   True       2m47s
```